backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -1,6 +1,6 @@
 :_content-type: ASSEMBLY
 [id="installing-oadp-mcg"]
-= Installing and configuring the OpenShift API for Data Protection with Multicloud Object Gateway
+= Installing and configuring the OpenShift API for Data Protection with OpenShift Data Foundation's Multicloud Object Gateway
```

Comment:
this doesn't seem to be related  to this PR @HagayVider1

Reply:
@HagayVider1 - please could you fix this as Amos requested? Then add the necessary to get it moved to OCP review.

Thanks